testdata/php/php8.1/plan.json

```diff
@@ -1,17 +1,25 @@
 {
+  "install_stage": {
+    "command": "composer install --no-dev --no-ansi",
+    "input_files": [
+      "."
+    ]
+  },
+  "build_stage": {
+    "command": ""
+  },
+  "start_stage": {
+    "command": "php -S 0.0.0.0:8080 -t public",
+    "input_files": [
+      "."
+    ]
+  },
   "dev_packages": [
     "php81",
-    "php81.Packages.composer"
+    "php81Packages.composer"
   ],
   "runtime_packages": [
     "php81",
-    "php81.Packages.composer"
-  ],
-  "install_stage": {
-    "command": "composer install --no-dev --no-ansi"
-  },
-  "build_stage": {},
-  "start_stage": {
-    "command": "php -S 0.0.0.0:8080 -t public"
-  }
-}
+    "php81Packages.composer"
```

Comment:
@mikeland86 I noticed this has changed. Is that expected?

Reply:
yeah, we now support php building.